lib/components/narrative/narrative-itineraries-header.tsx

```diff
@@ -35,6 +35,18 @@ const ItinerariesHeaderContainer = styled.div<{ showHeaderText: boolean }>`
   margin-left: ${(props) => (props.showHeaderText ? 'inherit' : 'auto')};
 `
 
+const SortResultsDropdown = styled(Dropdown)`
+  button {
+    border: none;
+    padding: 3px 7px;
+    line-height: 20px;
+  }
```

Comment:
If we're going to override the default styling of the dropdown in every place we're using it, then we should change the package in otp-ui. But personally, I prefer the padding and line heights that we have set in otp-ui, so I think we can just remove the border here and leave the other dropdowns unchanged.